app/src/components/common/PageHeader.tsx

```diff
@@ -43,7 +44,7 @@ const Styled = {
 
 interface Props {
   title: ReactNode;
-  onBackClick?: () => void;
+  showBackButton?: boolean;
```

Comment:
As we can use `useNavigate` hook here, refactored to have the navigation helper function defined in just this component instead of all components that use this component and want a back button.